tests/testthat/test-renderer4-prostateLasso.R

```diff
@@ -30,13 +30,11 @@ viz.no.time <- list(
       xmin=arclength.click-rect.width,
       xmax=arclength.click+rect.width,
       id=paste0("arclength", round(arclength.click, 1)*10),
-      clickSelects.variable="arclength",
-      clickSelects.value=arclength.click,
-      showSelected.variable="arclength",
-      showSelected.value=arclength.show,
       key=ifelse(
         arclength.click==arclength.show, 1,
         paste(arclength.click, arclength.show))),
+      clickSelects=c(arclength="arclength.click"),
+      showSelected=c(arclength="arclength.show"),
       alpha=0.5,
       data=addY(tallrect.all, "error")),
   res=ggplot()+
```

Comment:
interesting example. in the old syntax we had 
```r
clickSelects.variable="arclength",
clickSelects.value=arclength.click,
showSelected.variable="arclength",
showSelected.value=arclength.show,
```
and you're right with your new syntax, our compiler will look for a column named arclength. Right now you're right the most direct translation of this example would be to just create a new "selector.name" column in the data table (every row has value "arclength").

Ideally  not right now, but one day as an optimization) we could add a special case for this, I added a TODO https://github.com/tdhock/animint2/commit/5437ab74eec197dc07c2861a2bccdd1da90c6c7e